cime_config/testdefs/testmods_dirs/clm/soil_layerstruct/README

```diff
@@ -1,5 +1,10 @@
 This test sets soil_layerstruct_userdefined to the dzsoi values of the
 predefined case 4SL_2m and checks for bfb same answers.
 
+This is done by
+(a) listing the test in (1) testlist_clm.xml (as all tests) and
+                        (2) config_tests.xml
+(b) creating the file .../cime_config/SystemTests/soilstructud.py
+named after the test in lower case
```

Comment:
@billsacks I summarized here what I did to create the new test type. Do you see anything wrong at this point? I have not tested, yet, because cheyenne's compute nodes are down.

Reply:
Great - it looks like you did everything that's needed here. You should now change the testmods directory to simply clm-default, since the `soil_layerstruct` testmods directory no longer does anything beyond default.

Also: Have you done the step of ensuring that the test fails if you introduce a deliberate error (either in the code or in the test's python script – e.g., changing the dz for one of the soil layers)? I view that as an important step when introducing a new test (either a new system test or a new unit test).

Reply:
Ok, done. Thank you for the reminder to perform this sanity check. I ran it and got:
FAIL SOILSTRUCTUD_Ld5.f10_f10_musgs.I2000Clm50BgcCropGs.cheyenne_intel.clm-default COMPARE_base_4SL_2m

Reply:
Perfect, thanks.